frontend/data.ts

```diff
@@ -27,7 +27,10 @@ import type { UserInfo } from "../shared/client-state";
 export type Data = Await<ReturnType<typeof createData>>;
 type Await<T> = T extends PromiseLike<infer U> ? U : T;
 
-export async function createData(rep: Replicache) {
+export async function createData(
+  rep: Replicache<typeof mutators>,
```

Comment:
This worked out nicer than I first thought... I was like sh*t, this is not going to work... If you only do `Replicache` (without type param) it does not recognize the mutators.

Reply:
Nice. Another way to do it is probably to capture the type of the constructed Replicache instance somewhere, kind of like how I was capturing the type of the returned `data` value.